armi/reactor/blocks.py

```diff
@@ -1174,6 +1174,39 @@ def isPlenumPin(self, c):
             return True
         return False
 
+    def setB10VolParam(self, heightHot):
```

Comment:
Might be helpful to indicate in the docstring that this parameter is a boolean.

Reply:
done